### ISSUE TITLE
dta Manager - context menu issue with keyboard navigation

### ISSUE THREAD

**Issue by joshrc3** [downthemall/downthemall#446]
Now when I Press the Menu Key or Shift+F10 shortcut to open the context menu, the context menu will appear; but before I get a chance to select an option with the arrow keys up/down, the context menu disappears.

**Comment by nmaier** [comment]
Right, the context menus (which I had to emulate in html, as I have no access in a webextension to construct "native" ones) currently do not support keyboard navigation. I should implement this.
